tests/e2e/util.go

```diff
@@ -1051,17 +1051,19 @@ func setResourceQuota(client clientset.Interface, namespace string, size string)
 	deleteResourceQuota(client, namespace)
 
 	existingResourceQuota, err := client.CoreV1().ResourceQuotas(namespace).Get(ctx, namespace, metav1.GetOptions{})
-	gomega.Expect(err).NotTo(gomega.HaveOccurred())
-	framework.Logf("existingResourceQuota name %s", existingResourceQuota.GetName())
-	requestStorageQuota := updatedSpec4ExistingResourceQuota(existingResourceQuota.GetName(), size)
-	testResourceQuota, err := client.CoreV1().ResourceQuotas(namespace).Update(
-		ctx, requestStorageQuota, metav1.UpdateOptions{})
-	gomega.Expect(err).NotTo(gomega.HaveOccurred())
-	ginkgo.By(fmt.Sprintf("ResourceQuota details: %+v", testResourceQuota))
-	// TODO: Add polling instead of static wait time and assert against the
-	// updated quota.
-	ginkgo.By(fmt.Sprintf("Sleeping for %v seconds", sleepTimeOut))
-	time.Sleep(sleepTimeOut * time.Second)
+	if !apierrors.IsNotFound(err) {
```

Comment:
please revisit this block, I don't the logic is correct here.

Reply:
This is change is related to https://bugzilla.eng.vmware.com/show_bug.cgi?id=2710308

Reply:
When a test bed deployed manually  it will have "infinite storage limit"  on the namespace, It will not be having a resource quota name with the same name as namespace.  In that case this particular method was failing.  

Here we are checking  is there any quota with the same name as namespace , if present it increases quota limit. 
If there is no resource quota with the same name as namespace , it means  it has infinite storage limit and hence skipping quota assignment .

Reply:
you are exiting at the line 1056, rest of the code will not run

Reply:
That code will exit  only if there is no resource-quota with the same name as namespace present. It wont enter inside the If condition .

If there is no error , and if there is quota with same name as namespace  it will enter the if  and execute the code as it is. 

Though there is  namespace quota present and any other error occurred it will fail in line number 1056